tests/src/ExpertExtender/testExpertExtenderExtension.js

```diff
@@ -8,15 +8,13 @@
 	valueview.tests = valueview.tests || {};
 	valueview.tests.testExpertExtenderExtension = {
 		all: function( constructor, getInstance ) {
-			this.constructor( constructor, getInstance );
-			this.destroy( constructor, getInstance );
-			this.init( getInstance );
+			this.constructor( constructor, getInstance() );
+			this.destroy( constructor, getInstance() );
+			this.init( getInstance() );
```

Comment:
I still think instantiation should happen in the `QUnit.test` calls. Consider that this code runs for all QUnit test runs, even when these tests are filtered out.